js/customize-concurrency.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-extra-parens */
+
+( function( api ) {
+	'use strict';
+
+	var component;
+
+	if ( ! api.Concurrency ) {
+		api.Concurrency = {};
+	}
+
+	component = api.Concurrency;
+
+	component.data = {};
```

Comment:
Please do `_.extend( component.data, _customizeConcurrency );` or something to that effect, and then you can just refer to `component.data.session_start_timestamp`